api.go

```diff
@@ -217,6 +217,7 @@ type (
 	StreamPull struct {
 		Source   string            `json:"source"`
 		Headers  map[string]string `json:"headers,omitempty"`
+		IsMobile bool              `json:"isMobile,omitempty"`
```

Comment:
Where/what for do we need this? I'd really love for us to not make any real use of this value but the hackity hack that the customer asked from us (`fps->0` when mobile lol).

If we need this on the actual video pipeline instead, it should be identified from the actual video stream instead which is the only real source of truth here (this metadata means nothing, I could say it's mobile and stream from desktop).

WDYT?

Reply:
That said: this apparent boolean is actually an `int` 😅 
https://github.com/livepeer/livepeer-com/blob/79a7c3b5d7ece02b95527f59cc93d3f77eff171d/packages/api/src/schema/api-schema.yaml#L556-L567

We still kept the boolean option in the schema, but they will only use the `int`. Yet another reason not to want this being used anywhere else, such a bad interface.

Reply:
To be honest I added `IsMobile` ONLY because catalyst-api didn't compile without that. Maybe we don't need it. Could you check?

We use it here: https://github.com/livepeer/catalyst-api/blob/06a224db8a77729488271e126e61bfafd65c7bc0/handlers/geolocation/geolocation.go#L297

Reply:
@victorges @gioelecerati what do you think?

Reply:
IsMobile isn't needed here and in catalyst-api.
It was added because we were supposed to handle the fix in there, hence the `todo handle` under that flag to redirect to Os with the fix. But we did otherwise, right?

I think we can remove it here and in catalyst-api